features/standard/SystemInfo.feature

```diff
@@ -20,7 +20,7 @@ Feature: System info verification
         | ezsystems/ezplatform-design-engine      |
         | ezsystems/ezplatform-http-cache         |
         | ezsystems/ezplatform-solr-search-engine |
-        | ezsystems/ezpublish-kernel              |
+        | ezsystems/ezplatform-kernel             |
```

Comment:
Change to Behat scenario, FYI @ezsystems/qa-team. See passing [package build](https://travis-ci.org/github/ezsystems/ezplatform-admin-ui/builds/664916181) and [meta repository build](https://travis-ci.org/github/ezsystems/ezplatform/builds/664823684)